src/main/java/qna/domain/DeleteHistories.java

```diff
@@ -0,0 +1,15 @@
+package qna.domain;
+
+import java.util.List;
+
+public class DeleteHistories {
+    private final List<DeleteHistory> deleteHistories;
+
+    public DeleteHistories(List<DeleteHistory> deleteHistories) {
+        this.deleteHistories = deleteHistories;
```

Comment:
Collection에서 제공하는 unmodifiableList를 활용해보는 건 어떨까요?
리스트에 대한 추가, 삭제를 막아주는 기능으로 일급 컬렉션도 좀 더 불변에 가까워질 수 있습니다 :)

```suggestion
        this.deleteHistories = Collections.unmodifiableList(deleteHistories);
```